tests/test_aird_filters.py

```diff
@@ -4,66 +4,86 @@
 """Tests for aird-filters applied after rendering a diagram"""
 import pytest
 
-from capellambse import MelodyModel, aird
+import capellambse
+from capellambse import aird
+from capellambse import model as _m
 
 COMP_PORT_FILTER_DIAG = "[LAB] Test Component Port Filter"
 EX_ITEMS_FILTER_DIAG = "[SDFB] Test ExchangeItem Filter"
-STD_ACTIVATED_FLTERS = [
-    "ModelExtensionFilter",
-    "hide.overlappedfunctional.chains.label.filter",
-    "hide.overlappedfunctional.chains.icon.filter",
-]
+DEFAULT_ACTIVATED_FILTERS = frozenset(
+    [
+        "ModelExtensionFilter",
+        "hide.overlappedfunctional.chains.label.filter",
+        "hide.overlappedfunctional.chains.icon.filter",
+    ]
+)
 EX_ITEMS_FLTER = "show.exchange.items.filter"
 NAME_AND_EX_ITEMS_FILTER = "show.functional.exchanges.exchange.items.filter"
 
 
-def test_diagram_has_activated_filters(model_5_2: MelodyModel) -> None:
-    diag = model_5_2.diagrams.by_name(EX_ITEMS_FILTER_DIAG)
+def test_diagram_has_activated_filters(
+    model_5_2: capellambse.MelodyModel,
+) -> None:
+    diag: _m.diagram.Diagram = model_5_2.diagrams.by_name(EX_ITEMS_FILTER_DIAG)
+
+    assert diag.filters == DEFAULT_ACTIVATED_FILTERS
+
 
-    assert diag.activated_filters == STD_ACTIVATED_FLTERS
+def test_add_activated_filter_to_diagram(
+    model_5_2: capellambse.MelodyModel,
+) -> None:
+    diag: _m.diagram.Diagram = model_5_2.diagrams.by_name(EX_ITEMS_FILTER_DIAG)
 
+    diag.filters.add(EX_ITEMS_FLTER)
 
-def test_add_activated_filter_to_diagram(model_5_2: MelodyModel) -> None:
-    diag = model_5_2.diagrams.by_name(EX_ITEMS_FILTER_DIAG)
+    assert EX_ITEMS_FLTER in diag.filters
 
-    diag.add_activated_filter(EX_ITEMS_FLTER)
 
-    assert diag.activated_filters == STD_ACTIVATED_FLTERS + [EX_ITEMS_FLTER]
+@pytest.mark.parametrize("filter_name", DEFAULT_ACTIVATED_FILTERS)
+def test_remove_activated_filter_on_diagram(
+    model_5_2: capellambse.MelodyModel, filter_name: str
+) -> None:
+    diag: _m.diagram.Diagram = model_5_2.diagrams.by_name(EX_ITEMS_FILTER_DIAG)
 
+    diag.filters.remove(filter_name)
 
-def test_remove_activated_filter_on_diagram(model_5_2: MelodyModel) -> None:
-    diag = model_5_2.diagrams.by_name(EX_ITEMS_FILTER_DIAG)
+    assert filter_name not in diag.filters
```

Comment:
This assertion can easily be made even stricter:

```suggestion
    assert diag.filters == DEFAULT_ACTIVATED_FILTERS - {filter_name}
```